Makefile

```diff
@@ -17,8 +17,8 @@
 # limitations under the License.
 
 extensions = \
-	r-bioconductor_3_10 r \
-	cuda-9.2 cuda-10.0-tensorflow-1.14
+	bioc3_10 r3.6.1 \
+	cuda9.2 cuda10.0-tf1.14
 
 DOCKER_PREFIX?=renku/singleuser
```

Comment:
we changed the prefixes as well:

`renku/renkulab`

where `renku/renkulab-renku[version]-[languageVersion]-[label]`
and label is the commit hash OR version-tag from the repo